backend/db/ddl-6-cset_members_items.jinja.sql

```diff
@@ -18,7 +18,7 @@ SELECT
     item."includeMapped",
     c.vocabulary_id,
     c.standard_concept
-FROM {{schema}}concept_set_members{{optional_suffix}} csm
```

Comment:
## The bugfix
@Sigfried Just had to remove the `{{optional_suffix}}` part from the tables that are being selected from. This suffix is only needed for the current table that is in the process of being created by the DDL.

I'm bewildered as to why this started at 10am today when the offending commit was yesterday (3pm I think).